### ISSUE TITLE
Replace seawater with GSW

### ISSUE THREAD

**Issue by bhlevca** [eawag-surface-waters-research/PyLake#1]
Sewawater was deprecated please replace seawater with GSW. I replaced it in all my code,

When using pylake I get this warning:
pylake.py:3: UserWarning: The seawater library is deprecated! Please use GSW library instead.

I am using Python version 3.12. However, I think that seawater itself is issuing the deprecating warning as the Python distribution does not have deprecation insight about its libraries.

**Comment by f-baerenbold** [comment]
Thanks, we will do this in the next update.

**Comment by f-baerenbold** [comment]
Actually, the seawater library is only used to compute the thermal expansion coefficient (alpha) and the heat capacity (cp) to compute surface buoyancy flux. I wonder whether it makes sense to use the seawater library here if we talk about lakes. There would also be an equation for freshwater in teospy. I guess it would make more sense to use the latter.

**Comment by f-baerenbold** [comment]
will be updated soon (see other issues)

**Comment by bhlevca** [comment]
I see many things have changed, but the `seawater` dependency is still there.

**Comment by f-baerenbold** [comment]
Yes, we will get rid of it in favor of more lake specific calculations/libraries (see the open issues).
